### PR TITLE
Update CHANGELOG for 9.3.0

### PR DESCRIPTION
**Pull request by onurctirtir**
Update CHANGELOG for citus community 9.3.0.

\- Any review is welcomed not to miss any pr's

### CODE REVIEW

**Comment by SaitTalhaNisanci** [comment]
Maybe you can try the instructions on https://github.com/citusdata/citus/blob/master/src/test/scripts/README.md so that we won't have merge conflict to enterprise master :D

**Comment by metdos** [comment]
@onderkalaci shall we add a description for https://github.com/citusdata/citus/pull/3692, probably to the top of the changelog?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,81 @@
+### citus v9.3.0 (May 6, 2020) ###
```
Comment (onderkalaci) [inline CHANGELOG.md:1]: I'd re-order things a bit, put more useful things to many users goes top such as

so, `Adds support for window functions on coordinator` should come first and continues with
`Improves shard pruning logic to understand OR-conditions`
`Prevents using an extra connection for intermediate result multi-casts`
`Adds propagation of  `ALTER ROLE .. SET` statements`
`Introduces a UDF to truncate local data after distributing a table`

and all crash fixes goes down. Also, it'd be nice to have all local execution related changes to be next to each other in the list.

Also, we seem to miss description on https://github.com/citusdata/citus/pull/3692, now I added. Can you get it  here? It should also be on the top block

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+
+* Fixes possible issues by invalidating the plan cache in `master_update_node`
+
+* Fixes timing issues which could be caused by changing system clock
+
+* Ignores pruned target list entries in coordinator plan
+
+* Improves `SIGTERM` handling of maintenance daemon
+
+* Improves dependency resolution by utilizing `pg_shdepend` too
```
Comment (onderkalaci) [inline CHANGELOG.md:67]: This seems too internal information to be on the release notes: `Improves dependency resolution by utilizing `pg_shdepend` too`.

**Comment by onurctirtir** [comment]
> Maybe you can try the instructions on https://github.com/citusdata/citus/blob/master/src/test/scripts/README.md so that we won't have merge conflict to enterprise master :D

Yeah you are totally right, after getting and approval, I will actually open another pr on enterprise to update the changelog there as well, that would fix

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,81 @@
+### citus v9.3.0 (May 6, 2020) ###
+
+* Adds `update_distributed_table_colocation` UDF to update colocation of tables
+
+* Adds propagation of  `ALTER ROLE .. SET` statements
```
Comment (hanefi) [inline CHANGELOG.md:5]: Two blank spaces in here. Here is the current line followed by the one with correct number oof spaces:

```
 * Adds propagation of  `ALTER ROLE .. SET` statements
 * Adds propagation of `ALTER ROLE .. SET` statements
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,81 @@
...
+
+* Allows table type to be used in target list
+
+* Avoids having multiple maintenance daemons active for a single database
+
+* Defers reference table replication to shard creation time
+
+* Enables joins between local tables and reference tables in transaction blocks
+
+* Fixes a bug occurs when creating remote tasks in local execution
```
Comment (hanefi) [inline CHANGELOG.md:29]: bug **that** occurs

**Comment by metdos** [comment]
I can't see this, "Add max_shared_pool_size to control total number of connections across sessions"

Do we need to put the description to the top? https://github.com/citusdata/citus/pull/3692

**Comment by JelteF** [comment]
@metdos good catch. I guess it was not added automatically because the `DESCRIPTION:` is missing a space between `DESCRIPTION:` and the actual description. @onurctirtir can you change the script to make sure next time this happens we don't have this issue?

**Comment by onderkalaci** [comment]
> I have not checked if anything is missing, but the ones that are there look good.

Same for me, but we're not planning to merge 9 commits , right ? :)

**Comment by metdos** [comment]
> @metdos good catch. I guess it was not added automatically because the `DESCRIPTION:` is missing a space between `DESCRIPTION:` and the actual description. @onurctirtir can you change the script to make sure next time this happens we don't have this issue?

I think Onder added the description after I added my comment, so not sure if it is a script problem.

**Pull request merged**